index.html

```diff
@@ -9,38 +9,194 @@
+        </div>
+        <div class="project-container">
+          <img class="project-image" src="images/project-4.jpg" alt="project image">
+          <h2 class="project-name">Project name goes here</h2>
+          <ul class="languages-list">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button" class="see-this-project-button">See this project<i class="fa-solid fa-arrow-right"></i></button>
+        </div>
+        <div class="project-container">
+          <img class="project-image" src="images/project-5.jpg" alt="project image">
+          <h2 class="project-name">Project name goes here</h2>
+          <ul class="languages-list">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button" class="see-this-project-button">See this project<i class="fa-solid fa-arrow-right"></i></button>
+        </div>
+        <div class="project-container">
+          <img class="project-image" src="images/project-6.jpg" alt="project image">
+          <h2 class="project-name">Project name goes here</h2>
+          <ul class="languages-list">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button" class="see-this-project-button">See this project<i class="fa-solid fa-arrow-right"></i></button>
+        </div>
+    </section>
+
+    <!-- ABOUT SECTION -->
+
+    <section class="about-myself-section">
+      <div class="about-container">
+        <header class="section-header projects-header">
+          <h1>About</h1>
+        </header>
+        <div class="about-myself-container">
+          <p class="descriptive-p">
+           Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum molestias, quod recusandae asperiores similique quos aperiam aspernatur, ipsam autem fuga minima libero dolor accusantium corporis possimus quaerat repellat harum distinctio!
+          </p>
+        </div>
+        <button class="cv-button" type="button">Get my CV <i class="fa-solid fa-download"></i></button>
+      </div>
+
+      <div class="about-content-grid">
+        <div class="languages-container color-box-container">
+          <h3 class="title-about-content">Languages
+          </h3>
+          <ul class="about-content-list">
+            <li>JavaScript (ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>  
+        </div>
+        <div class="frameworks-container color-box-container">
+          <h3 class="title-about-content">Frameworks
+          </h3>
+          <ul class="about-content-list">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+            <li>Selenium</li>
+          </ul>  
+        </div>
```

Comment:
You asked for help on how to add those images!  Download them from your Figma template, then include them. Use this if you need help with positioning:

https://stackoverflow.com/questions/54079055/css-position-image-on-bottom-right-corner